src/lib/MongoDatabase.ts

```diff
@@ -16,8 +16,9 @@ export class MongoDatabase extends Database {
     new Promise(resolve => {
       // Connect to mongo
       console.log('MongoDatabase connect: ' + this.location);
-      this.client = new mongoDB.MongoClient(this.location);
-      this.database = this.client.db(this.db_name);
-      return resolve(this.client.connect());
```

Comment:
To connect using the default admin mongo credentials, reference the PIMS repository. The connection function should look somewhat like the following. Make sure to also add environment variables to the auth credentials and url scheme so that these can be configured easily within the docker environment if need be. See below:

```
const mongoHost = process.env.MONGO_HOSTNAME ? process.env.MONGO_HOSTNAME : 'mongodb://localhost:27017/pims'

  await mongoose.connect(mongoHost, {
      authSource: 'admin',
      'user': process.env.MONGO_USERNAME ? process.env.MONGO_USERNAME : 'rems-admin-pims-root',
      'pass': process.env.MONGO_PASSWORD ? process.env.MONGO_PASSWORD : 'rems-admin-pims-password',
  });
```